docs/docs/development/index.md

```diff
@@ -0,0 +1,90 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+Thanks for your interest in our project. Contributions are always welcome!
+
+We are committed to fostering a welcoming, respectful, and harassment-free
+environment. Be kind!
+
+If you have questions, ideas or want to report a bug, feel free to
+[open an issue](https://github.com/DSD-DBS/capella-collab-manager/issues). Or
+go ahead and
+[open a pull request](https://github.com/DSD-DBS/capella-collab-manager/pulls)
+to contribute code. In order to reduce the burden on our maintainers, please
+make sure that your code follows our style guidelines.
+
+## Setup of a local Development Environment
```

Comment:
```suggestion
## Setup of a local development environment
```

Reply:
In the last Collaboration Manager planning we agreed to use the Chicago Manual of Style, which says: 

> Always capitalize "major" words (nouns, pronouns, verbs, adjectives, adverbs, and some conjunctions).

My interpretation is that `Development Environment` is correct.